stable-diffusion/anaconda-project.yml

```diff
@@ -0,0 +1,120 @@
+  # include "notebook" or "dashboard".
+  deployments:
+    # Will be deployed at {projname_with_hyphens}-notebook.pyviz.demo.anaconda.com
+  - command: notebook
+    # Will be deployed at {projname_with_hyphens}.pyviz.demo.anaconda.com
+  - command: dashboard
+      # [optional] Set the AE5 container resource profile.
+      # Options include: "default", "medium" (default), "large"
+    resource_profile: medium
+      # [optional] Automatically start the deployment on AE5 when a PR
+      # modifying the project is merged. Default is true.
+    auto_deploy: true
+
+  # to build the website (e.g. too long or require too much data).
+  # This indicates the system not to run them.
+  skip_notebooks_evaluation: false
+  # For projects that get their data either by:
+  # - automatically generating it
+  # - providing indications
+  no_data_ingestion: false
+
+# required: (needed internally)
+user_fields: [examples_config]
+
+# required: list of the channels needed to solve the environment
+channels:
+- pyviz
+- defaults
+- conda-forge
+  # To remove the default channel, if need be.
+# - nodefaults
+
+# required: list of the direct dependencies needed to run the project.
+# Good practice is to pin their minimum version (>=) to indicate the
+# version with which the project has been created. Updates to the project
+# should updates these pins, if relevant.
+packages: &pkgs
+  # required: must be installed 
+- notebook 
+  # then list all your dependencies, including Python itself
+- python >=3.10,<3.11.0a0
+- panel >=0.14.2
+- diffusers 
+- transformers
+- pytorch 
+- ftfy
+
+dependencies: *pkgs
+
+# required: define the commands the project user may run (e.g. anaconda-project run notebook)
+commands:
+  # if you intend to run notebooks, set the `notebook` command
+  notebook:
+    # reference a single notebook in your project
+    notebook: stable-diffusion.ipynb
+    # shortcut to open all the notebook
+    # notebook: .
+  # if you intend to deploy a panel/lumen app, set the `dashboard` command
+  dashboard:
+    # the `--rest-session-info --session-history -1` options are required
+    unix: panel serve --rest-session-info --session-history -1 stable-diffusion.ipynb
+    supports_http_options: true
+  # required: a project must have a test command that references the 'test' env_spec
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
+  # required: a project must have a lint command that references the 'test' env_spec
+  lint:
+    unix: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+# required
+env_specs:
+  default: {}
+  test:
+    packages: &testpkgs
+    - nbsmoke
+    - pytest
+    dependencies: *testpkgs
+
+
+# optional environment variables
+variables: {}
+
+# optional downloads
+downloads: {}
+platforms:
+- linux-64
+- osx-64
+- win-64
```

Comment:
Only tested on linux w/ GPU

Reply:
Just FYI. Pretty sure most of the examples here have not been tested on all platforms. I'd guess that they were built on a Mac and then somewhat tested on the CI on Linux. The GPU is definitely an issue for the CI.

Reply:
Removed osx-64 and win-64 for now.

Reply:
Using `env-spec` to build custom env for M1, see comment here: https://github.com/pyviz-topics/examples/pull/254/commits/7d30c94c4bab14de000b87f7747eb67a6a94e1c6#r1119620931